### PR TITLE
Oct converter added to default image builders

### LINKED ISSUE #4: Add support for .e2e and .fds
See https://github.com/comic/grand-challenge.org/issues/1689

### PR DESCRIPTION
**Pull request by amickan**
OCT converter added to default image builders. 
- Supports .e2e, .fds and .fda images. 
- Extracts OCT as well as fundus image. 
- Next to the scans themselves, eye laterality and voxel spacing is extracted.

We added tests for the new builder and image formats. 
We do not yet have a public .e2e file, so the tests for .e2e are currently disabled. 

Closes #4

### CODE REVIEW

**Comment by jmsmkn** [comment]
Could you find smaller test images? As-is, this will make cloning and testing slow.

File: panimg/image_builders/oct.py [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+            oct_slice_size=oct_slice_size,
+            eye_choice=eye_choice,
+        )
+    for image in fundus_images:
+        eye_choice = LATERALITY_TO_EYE_CHOICE[image.laterality]
+
+        if file.suffix != ".e2e":
+            img_array = image.image.astype(np.uint8)
+            img_array = img_array[:, :, ::-1]
+            is_vector = True
```
Comment (jmsmkn) [inline panimg/image_builders/oct.py:64]: I'm not too sure if we can rely on checking the suffix here, maybe we need to try loading it as e2e and then adding the attributes we need for later. Also unsure about why we do the type conversion and invert the last axis.

File: panimg/image_builders/oct.py [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+        fda_img = FDA(file)
+        oct_slice_size = extract_slice_size(img=fda_img)
+        return (
+            [fda_img.read_oct_volume()],
+            [fda_img.read_fundus_image()],
+            oct_slice_size,
+        )
+    elif file.suffix == ".e2e":
+        e2e_img = E2E(file)
+        # TODO Document these size choices
```
Comment (jmsmkn) [inline panimg/image_builders/oct.py:161]: @amickan Could you add this comment?

Comment (amickan) [reply panimg/image_builders/oct.py:161]: will do!

File: panimg/image_builders/oct.py (hunk 1 above)
Comment (jmsmkn) [reply panimg/image_builders/oct.py:64]: Ok! Now that we have included the oct converter in contrib we can make those fixes later, but not in this PR.

**Comment by jmsmkn** [comment]
Do you have an e2e file that we can include in the test suite?

**Review by jmsmkn — APPROVED** [review]
LGTM!

**Pull request merged**